website/docs/developer/research/research-lambda-container.md

```diff
@@ -148,6 +148,152 @@ Based on this research, it looks like using lambda as containers is possible whe
 
 If the above issues are solved, then implementing lambda as container is recommended. One way to use private ECR repo is to use a script or terraform code if possible that will create the ECR repo and then build, tag and push the container image to that repo. One that is done, update the terraform lambda modules and deploy the lambda. An additional [card](https://bugs.earthdata.nasa.gov/browse/ORCA-375) has been created to look into this way.
 
+
+### Modifying get_current_archive_list and perform_orca_reconcile
+
+The Orca Internal Reconciliation workflow lambdas require an alternative approach.
+- The maximum time limit for lambdas is 15 minutes. These lambdas may take a significant amount of time, and should not be subject to this limitation.
+
+- Code changes
+  - Merge `get_current_archive_list` and `perform_orca_reconcile` into one codebase.
+  - Wrap functionality in a loop that will process the internal-report queue until no entries remain.
+  - Since ECS does not support timeout, create an overarching timing mechanic that exits if an infinite loop occurs while processing a queue entry.
+    - Alternatively, a side-program could manually stop the task if it exceeds its' time limit.
+    - Remember that in addition to processing time, Aurora Serverless can take up to 5 minutes to spin up.
+  - Raise the internal-report queue's `visibility_timeout_seconds` to the expected timeout.
+  - Environment variables can be passed in at task definition, or when the task is run. The former should be sufficient.
+
+- Use an alternative Dockerfile. The example below packages two code files into a lightweight python container with a `CMD` to run the main file.
+  ```yaml
+  FROM python:3.8-slim-buster
+
+  WORKDIR /src
+
+  # Copy function code
+  COPY main.py main.py
+  COPY sqs_library.py sqs_library.py
+
+  # Install the function's dependencies using file requirements.txt
+  # from your project folder.
+
+  COPY requirements.txt  .
+  RUN  pip3 install -r requirements.txt
+  COPY . .
+
+  CMD ["python", "main.py"]
+  ```
+- Follow prior instructions up to the Terraform deployment. Use the following Terraform instead of the pervious example.
```

Comment:
"previous" spelling.

I use Code Spell checker extension in vscode which is helpful in detecting wrong spellings.

Reply:
Fixed.